webv2/web.go

```diff
@@ -1173,6 +1175,49 @@ func getReportFile(w http.ResponseWriter, r *http.Request) {
 	w.Write([]byte(reportAbsPath))
 }
 
+func getDStructuredReport(w http.ResponseWriter, r *http.Request) {
```

Comment:
what does D for here in getDStructuredReport?

Reply:
> what does D for here in getDStructuredReport?

D refers to downloadable; kept it because there existed one getStructuredReport function before

Reply:
add a comment over these methods to explain the same

Reply:
> add a comment over these methods to explain the same

Done